### DISCUSSION TITLE
v0.6.6

### DISCUSSION THREAD

**Discussion by AIRLegend** [AIRLegend/aitrack#158] [General]
# Improvements

- IPv6 support for the Remote IP address
- Improve face detection. Now it runs faster with a **5x speedup** (lowers CPU consumption).
- Improvement of face detection in low light conditions.
- Improvement for the head position solver (improves yaw insensitivity and occasional extreme output values).
- (Advanced users only) You can now tweak the multithreading config of the app via `prefs.ini` (*this is temporary until a proper GUI is made*).

# Solved bugs

- Internal logit function could make facial landmarks "explode" under certain conditions. This release fixes that problem.
- Rotation (yaw, pitch and roll) values are now clipped to a max amount of +/-90 degrees.

# Acknowledgments
Thaks to @Lignar and @searching46dof  for their kind and nice contributions! 

Keep in mind this version is an alpha and bugs/malfunctions could be present. In case you encounter any issue, please, post it  on the [issues section](https://github.com/AIRLegend/aitracker/issues) or join our discord server to report it.

<a href="https://discord.gg/3XHY4xSDGv">
<img src="https://discord.com/assets/ff41b628a47ef3141164bfedb04fb220.png" width="90px"/>
</a><br/>
Join the server!

<hr /><em>This discussion was created from the release <a href='https://github.com/AIRLegend/aitrack/releases/tag/v0.6.6-alpha'>v0.6.6</a>.</em>

**Comment by BradleyGother** [comment]
v6,6 Not starting in windows 10. It has IP address box selected as default & cannot start using USB connection. 6.5 works, but high CPU usage compared to 6.3 which I use now.

**Comment by ghost** [comment]
First of all face detection in 0.6.6 is much better than 0.6.5! Good work!

I don't know why, but as I am facing the camera straight, I've got roll 90 degrees (in both versions). I guess the roll limit 90 degree now behaves so that I can roll my head only in one direction and it is detected and when rolling in another direction, the values is capped at 90 degrees and the roll in one direction is working but it isn't working in the opposite direction.

I've tried the same with 0.6.5 and there is the roll till 180 degree. And as I roll my head both sides, I get values above 120 in one direction and below 70 in the other one.

There should be maybe somewhere a switch to "reset" the roll to normal position 0 degree? Why is rotated 90 degrees in the beginning anyway?

I am using C270.
